playpen/coverage/coverage_hook

```diff
@@ -200,6 +206,75 @@ def _patch_entry_module(install):
                 file.writelines(lines)
 
 
+def _patch_pulp_resource_manager_service(install):
+    """Patch the pulp_resource_manager.service file.
+
+    :param install: boolean that indicates if the patch should be installed or
+        uninstalled, ``True`` or ``False`` respectively.
+    """
+    if install:
+        if _DISTRIBUTION == 'redhat' and _DISTRIBUTION_MAJOR_VERSION == '6':
+            subprocess.check_call([
+                'sed', '-i',
+                '/^VERSION=/ a CELERYD_OPTS="-P solo"',
+                '/etc/rc.d/init.d/pulp_workers',
+            ])
+        else:
+            subprocess.check_call([
+                'sed', '-i',
+                r's/\(--heartbeat-interval=[0-9]\+\).*$/\1 -P solo/',
```

Comment:
Since we're editing a unit file that's tightly integrated with Pulp itself, I definitely see why this line of code may need to be frequently updated. That said, I still have a big question mark floating over my head.

Here's some content from  `pulp_resource_manager.service` file:

```
ExecStart=/usr/bin/celery worker -A pulp.server.async.app -n resource_manager@%%h\
          -Q resource_manager -c 1 --events --umask 18 --pidfile=/var/run/pulp/resource_manager.pid\
          --heartbeat-interval=30
```

Let's imagine that the above was updated to:

```
ExecStart=/usr/bin/celery worker -A pulp.server.async.app -n resource_manager@%%h\
          -Q resource_manager -c 1 --umask 18 --pidfile=/var/run/pulp/resource_manager.pid\
          --heartbeat-interval=30 --events
```

Wouldn't the current regex be problematic, because it would drop the `--events` flag?

```
s/\(--heartbeat-interval=[0-9]\+\).*$/\1 -P solo/
```

Wouldn't something like this make the regex more resilient?

```
s/\(--heartbeat-interval=[0-9]\+\)/\1 -P solo /
```